python/lsst/ts/externalscripts/comcam/make_comcam_bias.py

```diff
@@ -0,0 +1,157 @@
+    a master bias calling the bias pipetask via OCPS.
+    """
+
+    def __init__(self, index=1):
+        super().__init__(
+            index=index,
+            descr="This class takes biases with LSSTComCam and constructs "
+                  "a master bias calling the bias pipetask via OCPS.",
+        )
+        self.comcam = ComCam(domain=self.domain, log=self.log)
+        self.ocps = salobj.Remote(domain=self.domain, name="OCPS")
+
+    @classmethod
+    def get_schema(cls):
+        schema = """
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: https://github.com/lsst-ts/ts_externalscripts/blob/master/python/lsst/ts/externalscripts/>-
+        comcam/make_comcam_bias.py
+        title: MakeComCamBias v1
+        description: Configuration for making a LSSTComCam master bias SAL Script.
+        type: object
+        additionalProperties: false
+        required: [instrument, inputCollections]
+        properties:
+            nBias:
+                type: integer
+                default: 1
+                description: number of biases to take
+
+            detectors:
+                type: tuple
+                items:
+                    type: integer
+                    minItems: 1
+                default: (0)
+                descriptor: Detector IDs
+
+            instrument:
+                type: string
+                enum: ["LSSTcComCam", "LATISS", "LSSTCam"]
+                descriptor: Camara name. Must be LSSTComca for now (future: also LATISS or LSSTCam).
+
+            inputCollections:
+                type: string
+                descriptor: Input collectiosn to pass to the bias pipetask.
+        """
+        return yaml.safe_load(schema)
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Parameters
+        ----------
+        config: `types.SimpleNamespace`
+            Configuration data. See `get_schema` for information about data
+            structure.
+        """
+        # Log information about the configuration
+
+        self.log.debug(
+            f"nBias: {config.nBias}, detectors: {config.detectors}, "
+            f"instrument: {config.instrument}. "
+        )
+
+        self.config = config
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script.
+        """
+        # Temporary number
+        metadata.duration = 10
+
+    async def run(self):
+        # Should ComCam and OCPS not be enabled here in the script?
+        # original notebook says: "When writing a script, the components
+        # should (probably) be enabled by a user."
+        await self.comcam.enable()
+        await salobj.set_summary_state(self.ocps, salobj.State.ENABLED,
+                                       settingsToApply="LSSTComCam.yaml")
+
+        # Take config.nBiases biases, and return a list of IDs
+        tempBiasList = await self.comcam.take_bias(self.config.nBias)
```

Comment:
we adopt `snake_case` for variables. I'd suggest `temp_bias_list`. But you can probably get rid of this and only use `exposures`...

```
exposures = tuple(await self.comcam.take_bias(self.config.nBias))
```